client/reader/stream/index.jsx

```diff
@@ -39,7 +39,7 @@ import CombinedCard from 'blocks/reader-combined-card';
 import fluxPostAdapter from 'lib/reader-post-flux-adapter';
 import config from 'config';
 import { keysAreEqual } from 'lib/feed-stream-store/post-key';
-import { isDiscoverBlog, isDiscoverFeed } from 'reader/discover/helper';
+import { combineCards, injectRecommendations, RECS_PER_BLOCK } from './utils';
```

Comment:
If its only used by two files within the same directory I don't think a 'config' file is necessary just yet